drivers/ethernet/phy/phy_mii.c

```diff
@@ -488,6 +488,7 @@ static const struct ethphy_driver_api phy_mii_driver_api = {
 #define PHY_MII_CONFIG(n)						 \
 static const struct phy_mii_dev_config phy_mii_dev_config_##n = {	 \
 	.phy_addr = DT_INST_REG_ADDR(n),				 \
+	.no_reset = DT_INST_NODE_HAS_PROP(n, no_reset),			 \
```

Comment:
Actually think this needs to be DT_PROP not HAS_PROP

Reply:
oh yes